pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -146,16 +146,6 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 	volumeNameMap := make(map[string]*v1.Volume)
 	networkNameMap := make(map[string]*v1.Network)
 
-	maxNumberOfDisksExceeded := len(spec.Domain.Devices.Disks) > arrayLenMax
```

Comment:
seems like you can delete const arrayLenMax right?

Reply:
ooops. thanks!

Reply:
done